docs/cookbook/adding-new-attribute-to-an-entity.md

```diff
@@ -37,8 +37,8 @@ It is a common modification when you need your e-commerce application and ERP sy
          */
         protected function __construct(BaseProductData $productData, ProductCategoryDomainFactoryInterface $productCategoryDomainFactory, array $variants = null)
         {
-            $this->extId = $productData->extId ?? 0;
             parent::__construct($productData, $productCategoryDomainFactory, $variants);
+            $this->extId = $productData->extId ?? 0;
```

Comment:
Hello @vitek-rostislav,

I have reviewed your changes and they are ok. I have one addition I like and I do in projects while extending parent classes. I have one empty line between calling parent method and my own code. It would be nice, if you could add it here and then you can send it to the BV. :)